vaccine_feed_ingest/runners/mn/gov/fetch.yml

```diff
@@ -0,0 +1,7 @@
+---
+state: mn
+# https://services.arcgis.com/9OIuDHbyhmH91RfZ/arcgis/rest/services/CovidVacLocations_view_prod/FeatureServer
+arcgis:
+- id: 18279cf4ab4740ae92708db437250056
+  layer_names:
+  - covidtestlocations # It says tests, but is actually vaccinations.
```

Comment:
😩

Reply:
This is also true of us/giscorps_vaccine_providers.  Sigh. :/